File: Hybrid_Task/Java/Dinner/ArmstrongNumber.java
import java.util.Scanner;

public class ArmstrongNumber {
public static void main(String[] args) {
Scanner input = new Scanner(System.in);

System.out.print("Enter a number: ");
int number = input.nextInt();

String num = Integer.toString(number);

Double sum = 0.00;

for(int count = 0; count < num.length(); count++) {
	char digits = num.charAt(count);
	int digit = Character.getNumericValue(digits);
	Double power = Math.pow(digit, 3);
	sum += power;
}

if(sum == number) System.out.print(number + " is an Armstrong number!");
else System.out.print(number + " is not an Armstrong number!");

}
}

File: Hybrid_Task/Python/Dinner/armstrong_number.py
number = input("Enter a number: ")

sum = 0
for digit in number:
	power = int(digit) ** len(number)
	sum += power

if sum == int(number):
	print(str(number) + " is an Armstrong number!")

if sum != int(number):
	print(str(number) + " is not an Armstrong number!")